### DISCUSSION TITLE
Separate CSS files

### DISCUSSION THREAD

**Discussion by alexander-lopez-s** [cristobal-lopez/my-portafolio#47] [General]
Hello @samirm00

I created a separate CSS file for the `responsiveness` of my portafolio because the `styles.css` file is already too long and it makes my Visual Studio slow.  Is this OK?

![Screenshot 2023-05-18 at 13 02 34](https://github.com/cristobal-lopez/my-portafolio/assets/97364283/1644329d-05f5-4f04-b300-5f45d9173fd8)

**Comment by sammou00** [comment]
Hi @cristobal-lopez , of course, it is. That is the right way.

**Reply by alexander-lopez-s** [reply]
Ok thank you 🙏
